notebook/README.md

```diff
@@ -1,31 +0,0 @@
-<!---
```

Comment:
Don't you think we should keep this section to match what we have in transformers (as done [here](https://github.com/huggingface/transformers/tree/main/notebooks) and [here](https://github.com/huggingface/transformers/blob/main/docs/source/en/notebooks.md)). I also realize that there is a spelling mistake (notebook instead of notebooks)

Reply:
Good idea! Done in [c84c098](https://github.com/huggingface/optimum/pull/395/commits/c84c09853b7564124035c2db65ab259d5d24c64b) and [3c7516d](https://github.com/huggingface/optimum/pull/395/commits/3c7516d201ac270dfdc9e9efb02b79eb3f5c6463)